test/bayes/bayesian_test.rb

```diff
@@ -1,7 +1,7 @@
 require File.dirname(__FILE__) + '/../test_helper'
 class BayesianTest < Test::Unit::TestCase
 	def setup
-		@classifier = ClassifierReborn::Bayes.new 'Interesting', 'Uninteresting'
+		@classifier = ClassifierReborn::Bayes.new 'Interesting', 'Uninteresting', language: 'en'
```

Comment:
It is the default but I think I wrote that test before there was a default, I'll clean it up.